module/x/gravity/types/genesis.go

```diff
@@ -134,6 +147,8 @@ func DefaultParams() *Params {
 		SlashFractionEthereumSignature:            sdk.NewDec(1).Quo(sdk.NewDec(1000)),
 		SlashFractionConflictingEthereumSignature: sdk.NewDec(1).Quo(sdk.NewDec(1000)),
 		UnbondSlashingSignerSetTxsWindow:          10000,
+		// this is in ethereum blocks. ethereum block time is ~12 seconds, about twice as long as Sommelier
+		EventVoteWindow: 5000,
```

Comment:
Should we not also be including `TxHistoryWindow` as defined in the genesis types file? Or is it never actually used and we should remove it?

Reply:
Ah, meant to remove it. Since pruning aligns with slashing now, it was redundant.